deployments/helm-chart-dos-arbitrator/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+name: nginx-appprotect-dos-arbitrator
```

Comment:
why are we adding the helm chart for the arbitrator to our repo?

Reply:
@pleshakov ^